zeppelin-zengine/src/test/java/org/apache/zeppelin/conf/ZeppelinConfigurationTest.java

```diff
@@ -41,7 +41,7 @@ public void cleanup() {
   }
 
   @Test
-  public void getAllowedOrigins2Test() throws MalformedURLException {
```

Comment:
I read the attached link and saw the suggestion of removing `public`. I'm fine with both, but prefer to keep things as-is to avoid introducing unnecessary diff.

Reply:
In general, I agree with you. Unnecessary diffs should be avoided. I myself use SonarLint https://www.sonarsource.com/products/sonarlint/ in my IDE to write clean code. Due to SonarLint and the IDE, many things are now marked that should be done differently with newer Java versions or JUnit versions. To prevent the markings from becoming too many and the important markings from being overlooked, I carry out such clean-up work automatically during development.
Unfortunately, this clean-up work distracts from the actual changes, which is why I'm using a separate PR this time.

Reply:
I'm fine with/without this change.